octodns/record/__init__.py

```diff
@@ -418,6 +418,7 @@ def __init__(self, _id, data):
             {
                 'value': d['value'],
                 'weight': d.get('weight', 1),
+                'up': d.get('up'),  # TODO: add validation on this field
```

Comment:
The meaning of `up` probably needs to be documented in https://github.com/octodns/octodns/blob/master/docs/dynamic_records.md. 

It seems like the concept exists in NS1 (`up`) & Azure (`endpoint_status`), not 100% sure `up` is the best name to use for it. Maybe `force_status` or something a bit more explicit? Open to suggestions, mostly don't feel like `up` describes the behavior sufficiently.

Reply:
Totally agreed that just `up` is not very good. I scratched my head for quite a bit on this.

We essentially want 3 states: force up, force down, and obey monitor. One was enums, but it can become unintuitive to use and complex to validate.

I thought of `force_status`, but `force_status=False` can mean "don't force, obey monitor" instead of "don't serve". Which then led me to also considering separate flags `force_up` and `force_down`, but we then end up with 4 possible value combinations of those flags which again makes it complex to build exhaustive logic as well as validate all combinations.

I really don't have any better ideas here for the flag name.

Reply:
I did like Route53's notion of `healthcheck=False` forcing up and `weight=0` for forcing down. But we again end up with 2 flags.

Reply:
I guess my first inclination would lean towards the enum route:

`status': (up|down|obey)` with obey being the implicit value/default.

That seems a little more explicit than `'up': (true|false)` and having that key missing meaning obey. 

Thoughts?

Reply:
Enum sounds intuitive for the user. Let me give it a shot.

Reply:
updated to an enum flag.